core/src/script/CGXP/plugins/Login.js

```diff
@@ -225,7 +235,8 @@ cgxp.plugins.Login = Ext.extend(gxp.plugins.Tool, {
      */
     addActions: function() {
         this.submitButton = new Ext.Button({
-            text: this.loginText,
+            text: this.loginFormText !== undefined
+                ? this.loginFormText : this.loginText,
```

Comment:
This doesn't work if ``loginFormText: ''`` (to skip the button text). This is why I have used a little more complicated syntax.